src/components/VideoPlayer/VideoPlayer.stories.mdx

```diff
@@ -0,0 +1,86 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import VideoPlayer from "./VideoPlayer";
+import { VideoPlayerTypes } from "./VideoPlayerTypes";
+import { action } from "@storybook/addon-actions";
+
+<Meta
+  title="VideoPlayer"
+  component={VideoPlayer}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=10968%3A3680",
+    },
+  }}
+  argTypes={{
+    attributes: { table: { disable: true } },
+    className: { table: { disable: true } },
+    modifiers: { table: { disable: true } },
+  }}
+/>
+
+# Video Player
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.23.2`   |
+
+<Description of={VideoPlayer} />
+
+The `VideoPlayer` component is used to embed a `Vimeo` or `YouTube` video player on a page. The component requires both the `videoType` and `videoId` props. You can extract the videoId from the video's URL. **IMPORTANT:** Do not try to enter the full video URL as the `videoId` value.
```

Comment:
With this lastest commit and this update about not entering a full video URL, I think it'll be good to check the `videoId` prop to verify it's not a full URL (possible ideas: either check that it's a certain pattern for vimeo and youtube, or check that "vimeo.com" and "youtube.com" are not in the string). If a full URL is passed instead of the video id, then render the error message that says the player has not been configured correctly.

Reply:
I added validation for videoId and also added a new check in the tests.